cmd/preview/list.go

```diff
@@ -72,16 +86,73 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		}
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
-	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
-	fmt.Fprintf(w, "Name\tScope\tSleeping\tLabels\n")
+	switch opts.output {
+	case "json":
+		if err != nil {
```

Comment:
@nitishfy the line `previewListOutput, err := getPreviewOutput(ctx, opts)` should go before you check for error nil

Reply:
Aah! sorry, didn't notice this thing. It's resolved now.